api/admin/search.go

```diff
@@ -42,37 +42,38 @@ type SearchResult struct {
 
 // SearchAsset represents the details of a single asset that was found.
 type SearchAsset struct {
-	PublicID      string              `json:"public_id"`
-	AssetID       string              `json:"asset_id"`
-	Folder        string              `json:"folder"`
-	AssetFolder   string              `json:"asset_folder"`
-	Filename      string              `json:"filename"`
-	DisplayName   string              `json:"display_name"`
-	Format        string              `json:"format"`
-	Version       int                 `json:"version"`
-	ResourceType  string              `json:"resource_type"`
-	Type          string              `json:"type"`
-	CreatedAt     time.Time           `json:"created_at"`
-	UploadedAt    time.Time           `json:"uploaded_at"`
-	Bytes         int                 `json:"bytes"`
-	BackupBytes   int                 `json:"backup_bytes"`
-	Width         int                 `json:"width"`
-	Height        int                 `json:"height"`
-	AspectRatio   float64             `json:"aspect_ratio"`
-	Pixels        int                 `json:"pixels"`
-	Tags          []string            `json:"tags"`
-	ImageMetadata ImageMetadataResult `json:"image_metadata"`
-	VideoMetadata MediaMetadataResult `json:"video_metadata"`
-	ImageAnalysis ImageAnalysis       `json:"image_analysis"`
-	URL           string              `json:"url"`
-	SecureURL     string              `json:"secure_url"`
-	Status        string              `json:"status"`
-	AccessMode    string              `json:"access_mode"`
-	AccessControl interface{}         `json:"access_control"`
-	Etag          string              `json:"etag"`
-	CreatedBy     SearchUser          `json:"created_by"`
-	UploadedBy    SearchUser          `json:"uploaded_by"`
-	LastUpdated   api.LastUpdated     `json:"last_updated"`
+	PublicID        string                `json:"public_id"`
+	AssetID         string                `json:"asset_id"`
+	Folder          string                `json:"folder"`
+	AssetFolder     string                `json:"asset_folder"`
+	Filename        string                `json:"filename"`
+	DisplayName     string                `json:"display_name"`
+	Format          string                `json:"format"`
+	Version         int                   `json:"version"`
+	ResourceType    string                `json:"resource_type"`
+	Type            string                `json:"type"`
+	CreatedAt       time.Time             `json:"created_at"`
+	UploadedAt      time.Time             `json:"uploaded_at"`
+	Bytes           int                   `json:"bytes"`
+	BackupBytes     int                   `json:"backup_bytes"`
+	Width           int                   `json:"width"`
+	Height          int                   `json:"height"`
+	AspectRatio     float64               `json:"aspect_ratio"`
+	Pixels          int                   `json:"pixels"`
+	Tags            []string              `json:"tags"`
+	ContextMetadata ContextMetadataResult `json:"context"`
```

Comment:
Let's call it `Context` for consistency with JSON, it's just a field name.
And you can reuse `ImageMetadataResult` type for it. It will not change in this version of the API.

Reply:
Hi, I have addressed the comments made between these revisions: https://github.com/cloudinary/cloudinary-go/compare/c7929f1c2c2240f5988a518e4d901fc36c1718a1..327f8fa549a384b1cd873346e7966d6aa8033c81 whenever you have time to take a look. 👍